src/test/java/products/CreateProducts.java

```diff
@@ -0,0 +1,54 @@
+package products;
```

Comment:
Pošto klasa sadrži testove, ona ne bi trebalo da bude u paketu u kome se nalaze modeli. Napravi paket koji se zove models i napravi paket koji se zove tests i razdvoji klase